### PR TITLE
Fixes #2 branch unification

### LINKED ISSUE #2: [spike] how much work to unify branches
Branch unification may save time later on, so it is worth spending max 2 hours on investigating what is involved and how difficult it is.

### PR DESCRIPTION
**Pull request by zeyus**
Completion of issue https://github.com/NLP4ALL/nlp4all/issues/2

After initial investigation, there were some filename issue (a file called `\`), it seems that there are no changes in the branches * `tf-idf-project-branch` and `confusion_matrix` so we can just merge them in, the `logistic-regression` branch will require more work and in fact may be something we should just re-implement once we've modularized the code...that's my suggestion, as you can see the merged branches only have one change each, that is the new file (`setup_users_BAD{x}.py`) which were formerly named `\`

### CODE REVIEW

File: setup_users_BAD.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+admin_role = Role(name='Admin')
+db.session.add(admin_role)
+student_role = Role(name='Student')
+db.session.add(admin_role)
+teacher_role = Role(name='Teacher')
+db.session.add(admin_role)
+db.session.commit()
+
+hp = bcrypt.generate_password_hash("Hermes_2014")
```
Comment (zeyus) [inline setup_users_BAD.py:24]: this is really really bad and we will have to address this and i will consider purging it from the entire repos history

**Review by zeyus — COMMENTED** [review]
just added a test comment

File: setup_users_BAD.py (hunk 1 above)
Comment (zeyus) [reply setup_users_BAD.py:24]: 🤦‍♀️🤦‍♀️🤦‍♀️🤦‍♀️🤦‍♀️

**Comment by emilroenn** [comment]
looks good

**Review by emilroenn — APPROVED** [review]
looks good

**Review by franciscoabenza — COMMENTED** [review]
all good

**Pull request merged**